tools/tests/dropdown/test.js

```diff
@@ -121,14 +132,17 @@ describe('Dropdown Tests', () => {
     afterAll(() => {
         const currentElement = document.querySelector('.dropdown-test-wrapper');
 
+        // Reset .options to avoid mixing the options re-added by the caching feature which messes up the tests.
+        currentElement.querySelector('.options').innerHTML = '';
+
         if (currentElement) {
             currentElement.parentElement.removeChild(currentElement);
         }

```

Comment:
Use the helper function from helpers if it is merged before you merge.

Reply:
Used and tests passed.